src/seedu/addressbook/data/person/Phone.java

```diff
@@ -2,15 +2,17 @@
 
 import seedu.addressbook.data.exception.IllegalValueException;
 
+
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.